Legal-Forms/Funerary Procedure and Last Will.txt

```diff
@@ -0,0 +1,33 @@
+[color=#9fed58]╔══════════════════╗
+                                ║███░███░░░░██░░░░░║
+                                ║░██░████░░░██░░░░░║
+                                ║░░█░██░██░░██░█░░░║
+                                ║░░░░██░░██░██░██░░║
+                                ║░░░░██░░░████░███░║
+                                ╚══════════════════╝[/color]
+                                            [head=3]NanoTrasen[/ head]
+                                     [head=3][color=#9fed58]Funeral Intructions[/color][/head]
+[bold]─────────────────────────────────────────[/bold]
+
+[head=2][italic]Upon untimely demise, ___[color=#5b97bc]NAME HERE[/color]___ requests the following:[/italic][/head] 
+
+[color=#aaaaaa][italic]choose one, items marked with * may not be availible in all stations and will be carried out by Central Command[/color][/italic]
+  -  (  ) Morgue
+  - 	(  ) Cremation
+  - 	(  ) Burial*
+  - 	(  ) Borgifying
```

Comment:
I think these items are supposed to be indented from the others, but ingame they are on the same indent
![image](https://github.com/user-attachments/assets/50eb8e69-b051-4651-a8fe-ff00c97ec7f4)